src/TabPane.js

```diff
@@ -133,6 +140,13 @@ let TabPane = React.createClass({
       : context.animation;
   },
 
+  getUnmountOnExit(props = this.props, context = this.context) {
```

Comment:
`getTransition` needs it because it gets called from `componentWillUpdate` with `nextProps` and `nextState`. It's not really needed here, though.